framework/include/base/ScalarCoupleable.h

```diff
@@ -182,6 +182,13 @@ class ScalarCoupleable
    */
   MooseVariableScalar * getScalarVar(const std::string & var_name, unsigned int comp);
 
+  /**
+   * Checks to make sure that the current Executioner has set "_is_transient" when old/older values
+   * are coupled in.
+   * @param name the name of the variable
+   */
+  void validateExecutionerType(const std::string & name, const std::string & fn_name) const;
```

Comment:
Your doc is out of date here. It would be useful to know what fn_name is. I had to look at the source and usage to figure it out. It's an internal function so I'm not too concerned.

Reply:
It is not out of date, I forgot to document the parameter. I will fix it...